package.json

```diff
@@ -3,6 +3,15 @@
   "version": "1.4.1",
   "description": "MagicMirror² Module that displays the current moon phase",
   "main": "MMM-MoonPhase.js",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/NolanKingdon/MMM-MoonPhase.git"
+  },
+  "keywords": [
+    "MagicMirror",
+    "Moon",
+    "Phase",
```

Comment:
Please remove the trailing comma at the end of line 13. That's not allowed in JSON files.